it/it-tests/src/test/java/it/organization/OrganizationMembershipTest.java

```diff
@@ -152,57 +151,58 @@ public void user_creating_an_organization_becomes_member_of_this_organization()
   @Test
   public void should_display_members_page() {
     String orgKey = createOrganization();
-    userRule.createUser("foo", "pwd");
-    userRule.createUser("bar", "pwd");
+    String userFoo = createUser("foo");
+    String userBar = createUser("bar");
```

Comment:
🐜  You might consider rearranging the order of the lines, to make it more obvious, that one of the users is not made a member.